.github/workflows/cypress.yml

```diff
@@ -1,42 +1,18 @@
 name: Cypress Tests
-on:
-  workflow_call:
-    secrets:
-      CYPRESS_TEST_USER_PASS:
-        required: true
+on: [deployment_status]
+
 jobs:
   Setup:
+    if: github.event.deployment_status.state == 'success'
```

Comment:
The 2 lines added here is all you need to have Cypress run after Heroku finishes deploying ✨